### PR TITLE
[CA-1241] Merge in develop branch with java11 changes

### PR DESCRIPTION
**Pull request by andy7i**
https://broadworkbench.atlassian.net/browse/CA-1241

This PR merges in the `develop` branch into the v2_workspaces feature branch. As far as I can tell, all changes are directly from the develop branch. There were 2 minor merge conflicts, one of which doesn't even show up in the diff (it is related to the clone workspaces tracing that I added in previously) and I added a comment for the other one. 

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [x] **Submitter**: Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [x] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] Tell your tech lead (TL) that the PR exists if they want to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -108,8 +108,10 @@ object Dependencies {
 
   val accessContextManager = "com.google.apis" % "google-api-services-accesscontextmanager" % "v1beta-rev55-1.25.0"
 
-  val workspaceManager = excludeGuavaJDK5("bio.terra" % "workspace-manager-client" % "0.13.0-SNAPSHOT")
-  val dataRepo = excludeGuavaJDK5("bio.terra" % "datarepo-client" % "1.0.44-SNAPSHOT")
+  def excludeJakartaActivationApi(m: ModuleID): ModuleID = m.exclude("jakarta.activation", "jakarta.activation-api")
+
+  val workspaceManager = excludeJakartaActivationApi("bio.terra" % "workspace-manager-client" % "0.16.0-SNAPSHOT")
+  val dataRepo = excludeJakartaActivationApi("bio.terra" % "datarepo-client" % "1.0.44-SNAPSHOT")
   val resourceBufferService = excludeGuavaJDK5("bio.terra" % "terra-resource-buffer-client" % "0.4.3-SNAPSHOT")
```
Comment (andy7i) [inline project/Dependencies.scala:115]: minor merge conflict here -- not really sure why since the only thing we really changed was adding line 115.

**Review by s-rubenstein — COMMENTED** [review]
Is there a reason not to do this as a rebase? I am worried that we are going to need to deal with all of these as merge conflicts later, when we try to go to develop, and we will have less context then.

File: project/Dependencies.scala (hunk 1 above)
Comment (s-rubenstein) [reply project/Dependencies.scala:115]: I think that's because the diff on the line was different in git.

**Comment by andy7i** [comment]
Yeah, I actually strongly prefer to do this as a merge:

* Doing it as a rebase would break the history for any existing branches off of v2_workspaces and force those branches to resync history.
* With a merge, I don't think we're introducing any additional merge conflicts down the line compared to a rebase. As a data point, Greg did a significant merge (which I believe was a merge and not a rebase) and none of those conflicts popped up again. (And as an aside, I've had some pretty frustrating experiences with rebasing where I needed to resolve the same merge conflict many times for multiple commits during the same rebase session)
* The benefits of sequential commit history that comes with rebasing will be lost when we squash and merge this PR. 

That being said, I'm open to hearing about advantages of rebasing that I haven't considered and going with that if that seems to be a better option.

**Review by gpolumbo-broad — APPROVED** [review]

**Comment by gpolumbo-broad** [comment]
I am definitely an advocate of merging over rebasing.  There's plenty of articles online that debate the merits of both and if it's a conversation we think we need to have, then we should have it.  

On a similar note, I think when I did my merge, I just did it on my local machine and plugged away at it until my tests passed.  So that's perhaps another team norm we can discuss is whether we thinking merges like this one require a PR or not?  I find this challenging to review because there are N other issues from the dev branch all lumped together into this one PR.  In order for me to be truly thorough in this review, I'd need to go look at each of those other PRs/issues that were merged to `develop` and understand those changes in order to verify that _this_ PR is right.  

Since we're just "catching up our long lived branch to the latest from `develop`", I'm actually comfortable skipping the PR so long as the merger has verified tests are passing.

**Review by s-rubenstein — APPROVED** [review]
Totally reasonable response, LGTM. Second Greg's point about not needing a review for this sort of thing though, since it is ultimately code that got a review already, and the context here is missing.

I do have suspicions we could still run into merge conflicts later since the PR is from a branch of yours into v2_workspaces, rather than from develop into v2_workspaces, but I haven't tested that assumption, so happy to trust that we are good. And, regardless, we're probably going to have messy merge conflicts to deal with later anyway.

**Pull request merged**